unlock-app/src/components/interface/checkout/Stepper.tsx

```diff
@@ -79,22 +81,26 @@ export interface StepItem {
 }
 
 interface StepperProps {
-  items: StepItem[]
-  position: number
   service: CheckoutService | UnlockAccountService
   disabled?: boolean
 }
 
-export const Stepper = ({
-  items,
-  position,
-  service,
-  disabled,
-}: StepperProps) => {
-  const index = items.findIndex((item) => item.id === position)
+export const Stepper = ({ service, disabled }: StepperProps) => {
+  const [state] = useActor(service)
+
+  const isUnlockAccount = service.id === 'unlockAccount'
+  // @ts-expect-error Property 'renew' does not exist on type 'UnlockAccountMachineContext'.
```

Comment:
I am not sure how to avoid these @searchableguy ?

Reply:
I think you need instance of or type of check to narrow down here.

Reply:
> 'CheckoutService' only refers to a type, but is being used as a value here.ts(2693)